docs/CONTRIBUTING-zh_TW.md

```diff
@@ -44,17 +44,17 @@
     * 不要提供短連結
 * 優先選擇使用 "current" 連結代替有 "version" 連結：使用 `http://example.com/dir/book/current/` 比使用 `http://example.com/dir/book/v1.0.0/index.html` 更好。
 * 如果一個連結存在過期的證書/自簽名證書/SSL問題的任何其他類型：
-  1. *replace it* ：如果可能的話，將其 *替換* 為對應的 `http` (因為在移動設備上接受異常可能比較複雜)。
-  2. *leave it* ：如果沒有http版本，但仍然可以通過https造訪連結，則在瀏覽器中添加異常或忽略警告。
-  3. *remove it* ：上述狀況以外則刪除掉它。
+    1. *replace it* ：如果可能的話，將其 *替換* 為對應的 `http` (因為在移動設備上接受異常可能比較複雜)。
+    2. *leave it* ：如果沒有`http`版本，但仍然可以通過`https`造訪連結，則在瀏覽器中添加異常或忽略警告。
+    3. *remove it* ：上述狀況以外則刪除掉它。
 * 如果一個連結以多種格式存在，請添加一個單獨的連結，並註明每種格式。
 * 如果一個資源存在於Internet上的不同位置
     * 使用最權威來源的連結(意思是原始作者的網站比編輯的網站好，比第三方網站好)。
     * 如果它們連結到不同的版本，你認為這些版本差異很大，值得保留，那麼添加一個單獨的連結，並對每個版本做說明(參考 [Issue #2353](https://github.com/EbookFoundation/free-programming-books/issues/2353) 有助於格式化問題的討論)。
 * 相較一個比較大的提交，我們更傾向於原子提交(通過添加/删除/修改進行一次提交)。在提交PR之前没有必要壓縮你的提交。(為了維護人員的方便，我們永遠不會執行這個規則)。
 * 如果一本書比較舊，請在書名中註明出版日期。
-* 包含作者的名字或適當的名字。中文版本可以用 “等” 縮短作者列表。
-* 如果一本書還没有完成，並且仍在編寫中，則需添加 “編寫中” 符號，參考 [下文](#in_process) 所述。
+* 包含作者的名字或適當的名字。中文版本可以用 “`等`” (“`et al.`”) 縮短作者列表。
+* 如果一本書還没有完成，並且仍在編寫中，則需添加 “`in process`” 符號，參考 [下文](#in_process) 所述。
```

Comment:
why un-translate this?

Reply:
Same reason like in greek doc (see https://github.com/EbookFoundation/free-programming-books/pull/6625#discussion_r791060297)